systemd/systemd.go

```diff
@@ -1432,6 +1437,7 @@ What={{.What}}
 Where={{.Where}}
 Type={{.Fstype}}
 Options={{join .Options ","}}
+LazyUnmount=yes
```

Comment:
I see all the other generate mount units have it, so it is fine. But I think one day we should remove all the `LazyUnmount` in all the generated units. I do not think they are needed. And if we have bugs, we should fix them another way.

Reply:
Right, I agree